CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Added `Group` to `compas.scene`.
 * Added `compas.geometry.Brep.cap_planar_holes`.
 * Added `compas_rhino.geometry.RhinoBrep.cap_planar_holes`.
+* Added `compas.geometry.angles.angle_vectors_projected`.
```

Comment:
Since the changelog is for "dev/end user consumption", I would use the 2nd-level import of this
```suggestion
* Added `compas.geometry.angle_vectors_projected`.
```